quint/io-cli-tests.md

```diff
@@ -928,7 +929,7 @@ rm out-itf-example*.itf.json
 
 <!-- !test in test itf -->
 ```
-output=$(quint test --out-itf='coin_{seq}_{test}.itf.json' \
+output=$(quint test --out-itf='coin_{seq}_{test}.itf.json' --verbosity=0 \
```

Comment:
Seems like the test command already had the "no suppressing output" behavior, so I think we don't need to change this one
```suggestion
output=$(quint test --out-itf='coin_{seq}_{test}.itf.json' \
```